zerver/tests/test_subs.py

```diff
@@ -2331,7 +2331,7 @@ def test_subscriber(self):
     def test_gather_subscriptions(self):
         # type: () -> None
         """
-        gather_subscriptions returns correct results with only 3 queries
```

Comment:
Note that the `3` in this comment was already out of sync (was really 4 already).